components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/Context.kt

```diff
@@ -79,3 +80,25 @@ fun Context.share(text: String, subject: String = getString(R.string.mozac_suppo
         false
     }
 }
+
```

Comment:
You could add a `var isMainProcess: Boolean? = null` here and only run the block in `isMainProcess()` once (if it is null).

Reply:
I was a bit afraid to do that, as `var isMainProcess: Boolean? = null` will be static and shared but two processes will have different heaps , and that will be OK. I updated the PR we the change!